src/components/Stats/PoolListItem/PoolListItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Grid, Typography, Box } from '@material-ui/core'
+import useStyle from './style'
+import { colors } from '@static/theme'
+
+interface IProps {
+  TVL?: string
+  volume?: string
+  fee?: string
+  displayType: string
+  symbolFrom?: string
+  symbolTo?: string
+  iconFrom?: string
+  iconTo?: string
+  tokenIndex?: number
+}
+
+const PoolListItem: React.FC<IProps> = ({
+  fee = '0',
+  volume = '0',
+  TVL = '0',
+  displayType,
+  symbolFrom,
+  symbolTo,
+  iconFrom,
+  iconTo,
+  tokenIndex
+}) => {
+  const classes = useStyle()
+  return (
+    <Grid>
+      {displayType === 'token' ? (
+        <Grid
+          container
+          classes={{ container: classes.container, root: classes.header }}
+          style={{ color: colors.white.main }}>
+          <Typography>{tokenIndex}</Typography>
+          <Grid className={classes.imageContainer}>
+            <Box>
+              <img src={iconFrom} />
+              <img src={iconTo} />
+            </Box>
+            <Box className={classes.symbolsContainer}>
+              <Typography>{symbolFrom}</Typography>/<Typography>{symbolTo}</Typography>
```

Comment:
you can use only 1 Typography component, and use <span> inside to change styles for symbolTo